docs/nydusd.md

```diff
@@ -319,64 +319,7 @@ or
   }
 }
 ```
-
-The `HttpProxy` backend also supports the `Proxy` and `Mirrors` configurations for remote usage like the `Registry backend` described above.
```

Comment:
The `HttpProxy` backend also supports the `Proxy` configurations for remote usage like the `Registry backend` described above.